Backpack-SwiftUI/TextArea/Classes/BPKTextArea.swift

```diff
@@ -87,35 +88,52 @@ public struct BPKTextArea: View {
     }
     
     public var body: some View {
-        ZStack(alignment: .topLeading) {
-            // Text editor
-            textEditorContent
-                .foregroundColor(.textPrimaryColor)
-                .padding(.vertical, TextEditorConstants.verticalPadding)
-                .padding(.horizontal, TextEditorConstants.horizontalPadding)
-            
-            // Placeholder
-            if let unwrappedPlaceholder = placeholder {
-                BPKText(unwrappedPlaceholder, style: .bodyDefault)
-                    .foregroundColor(.textSecondaryColor)
-                    .allowsHitTesting(false)
-                    .padding(.vertical, PlaceholderConstants.verticalPadding)
-                    .padding(.horizontal, PlaceholderConstants.horizontalPadding)
-                    .opacity(value.isEmpty ? 1 : 0)
+        HStack(alignment: .top, spacing: BPKSpacing.none) {
+            ZStack(alignment: .topLeading) {
+                // Text editor
+                textEditorContent
+                    .foregroundColor(.textPrimaryColor)
+                    .padding(.vertical, TextEditorConstants.verticalPadding)
+                    .padding(.horizontal, TextEditorConstants.horizontalPadding)
+                
+                // Placeholder
+                if let unwrappedPlaceholder = placeholder {
+                    BPKText(unwrappedPlaceholder, style: .bodyDefault)
+                        .foregroundColor(.textSecondaryColor)
+                        .allowsHitTesting(false)
+                        .padding(.vertical, PlaceholderConstants.verticalPadding)
+                        .padding(.horizontal, PlaceholderConstants.horizontalPadding)
+                        .opacity(value.isEmpty ? 1 : 0)
+                }
             }
-            
-            // Border
-            RoundedRectangle(cornerRadius: BorderConstants.cornerRadius)
-                .strokeBorder(.lineColor, lineWidth: BorderConstants.lineWidth)
+            accessory
+                .padding([.top, .trailing], .md)
         }
         .background(.surfaceDefaultColor)
         .clipShape(
             RoundedRectangle(cornerRadius: BorderConstants.cornerRadius)
         )
+        .outline(state.borderColor, cornerRadius: BorderConstants.cornerRadius)
         .frame(minHeight: frameHeight)
         .accessibilityLabel(accessibilityLabelText)
     }
     
+    @ViewBuilder
+    private var accessory: some View {
+        if let icon = state.icon {
+            BPKIconView(icon.icon)
+                .foregroundColor(icon.color)
+                .accessibilityHidden(true)
+        } else {
+            EmptyView()
+        }
```

Comment:
you dont need the else when this is a view builder! 
```suggestion
    private var accessory: some View {
        if let icon = state.icon {
            BPKIconView(icon.icon)
                .foregroundColor(icon.color)
                .accessibilityHidden(true)
        }
```